include/irr/asset/format/EFormat.h

```diff
@@ -1007,12 +1014,27 @@ namespace asset
                 blockByteSize(getTexelOrBlockBytesize(format))
             {}
             
+            //! It converts input texels strides to compute a true extent of an image
+            /*
+                The true extent is a dimension of stride in texels or in blocks, depending
+                of what are you dealing with.
+                @param coord it's a dimension in texels.
+            */
 
 			inline auto convertTexelsToBlocks(const core::vector3du32_SIMD& coord) const
 			{
 				return (coord+maxCoord)/dimension;
 			}
 
+            //! It converts input texels strides to compute length-block size
+            /*
+                Inverse of convertTexelsToBlocks. Since your format may be
+                block compressed, you can gain your length in block-size style.
+                For instance - given a BC texetur 4x4 having 64 texels, you can
+                compute how many blocks is in a row, you will use this function to
+                do that - the block-size will be 16 in that case.
```

Comment:
nope, it will return 64 for 64 with a 4x4

But for a 1,2,3 will return 4, it rounds up.. its quite useful to determine actual mipmap sizes of BC compressed mip map levels or textures not aligned to block size (gives you the size of the overlap)

Reply:
adjusted in following commit